Sources/Overload/OvEditor/src/OvEditor/Core/Context.cpp

```diff
@@ -154,6 +154,7 @@ OvEditor::Core::Context::Context(const std::string& p_projectPath, const std::st
 	ServiceLocator::Provide<OvAudio::Core::AudioPlayer>(*audioPlayer);
 	ServiceLocator::Provide<OvCore::Scripting::ScriptEngine>(*scriptEngine);
 	ServiceLocator::Provide<OvEditor::Utils::TextureRegistry>(*textureRegistry);
+	ServiceLocator::Provide<OvUI::Core::UIManager>(*uiManager);
```

Comment:
Wondering if we need to expose the `UIManager`, since you can get it from anywhere in `OvEditor` with `EDITOR_CONTEXT(uiManager)`. Services are mostly for the `OvCore` module to access other systems, without a global context